src/tree/LocalRootTreeItemBase.ts

```diff
@@ -72,6 +74,10 @@ export abstract class LocalRootTreeItemBase<TItem extends ILocalItem, TProperty
         return workspace.getConfiguration(`${configPrefix}.${this.treePrefix}`);
     }
 
+    private get autoRefreshEnabled(): boolean {
+        return window.state.focused && LocalRootTreeItemBase.autoRefreshViews === true;
```

Comment:
Nit, we rarely do `=== true`
```suggestion
        return window.state.focused && LocalRootTreeItemBase.autoRefreshViews;
```